### PR TITLE
Update issue templates with post-graduation checklist

### PR DESCRIPTION
**Pull request by tobie**
Depends on https://github.com/openjs-foundation/cross-project-council/pull/386.

### CODE REVIEW

File: ISSUE_TEMPLATE/01-project-post-graduation-checklist-template.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+---
+
+## Project Post-graduation Checklist - ${Project Name}
+
+<!--Copied from ../../PROJECT_PROGRESSION.md on 2019-10-31 -->
+
+**Stage: ${Project Stage}
+
+- [ ] Transfer official domains to OpenJS Foundation
```
Comment (ljharb) [inline ISSUE_TEMPLATE/01-project-post-graduation-checklist-template.md]:
```suggestion
- [ ] Transfer previously indicated official domains to OpenJS Foundation, if any
```

File: ISSUE_TEMPLATE/01-project-post-graduation-checklist-template.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+- [ ] If relevant, implement the appropriate tool to enforce CLA or DCO
+- [ ] Transfer trademarks to the OpenJS Foundation
```
Comment (ljharb) [inline ISSUE_TEMPLATE/01-project-post-graduation-checklist-template.md]:
```suggestion
- [ ] Transfer previously indicated trademarks to the OpenJS Foundation, if any
```

**Comment by tobie** [comment]
@ljharb, I shouldn't have merged those here, actually. Those changes should me made in https://github.com/openjs-foundation/cross-project-council/pull/386 first. Could you please suggest them there too?

**Comment by tobie** [comment]
Now that openjs-foundation/cross-project-council#386 has been landed, can we just land this?

**Pull request merged**